.vscode/settings.json

```diff
@@ -6,6 +6,8 @@
       "go.toolsEnvVars": {
             "GOFUMPT_SPLIT_LONG_LINES": "on"
       },
+      "go.lintTool": "golangci-lint",
+      
```

Comment:
I removed the "--fast" option they suggested because it really seemed to catch less things